### PR TITLE
Remove hash rocket operator

### PR DESCRIPTION
**Pull request by mbrandonw**
We've been having some trouble with type inference and the `=>` operator since it's used in both `HTML` and `CSS`. the operator makes more sense in the `HTML` package due to how it's used, whereas in `CSS` we can either use `property <| value` (since it's just function application) or even `property(value)`.

I've gone with the former for now, but we can easily change. the one downside to `<|` is that it looks too similar to `|<` which is what we use now to bind a `Css` value to a `CssSelector` value. that operator should change anyway tho...

### CODE REVIEW

File: Sources/CSS/Reset.swift [hunk 1]
```diff
@@ -48,12 +48,12 @@ private let allReset: Css = (
     | ul
     | video
 ) |< (
-  margin(all:) => "0"
-    <> padding(all:) => "0"
-    <> fontSize => pct(100)
-    <> fontFamily => .inherit
-    <> fontStyle => .inherit
-    <> fontWeight => .inherit
+  margin(all:) <| "0"
```
Comment (stephencelis) [inline Sources/CSS/Reset.swift]: Makes me wonder what it would look like sans-operator in most cases:

``` swift
margin(all: "0")
  <> padding(all: "0")
  <> fontSize(pct(100))
  <> fontFamily(.inherit)
  <> fontStyle(.inherit)
  <> fontWeight(.inherit)
```

Maybe a lil busy? But not bad!

Comment (mbrandonw) [inline Sources/CSS/Reset.swift]: yeah I think we could def just do that! will leave for another pr
